eva.xsh

```diff
@@ -75,7 +78,7 @@ def run_task(task_name, times):
     cd @(config.workspace)
     env = {}
     exec('from tasks.%s.continuous_evaluation import tracking_kpis'
-             % task_name, env)
+             % task_name.split('-')[0], env)
```

Comment:
why `-` here?

Reply:
import kpis from base model  
base model is as  resnet50   
models are like resnet50-v100-cardn  which are splited by '-'